integration_test/third_party_apps_test/applications/dcgmv1/metadata.yaml

```
@@ -0,0 +1,161 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+app_url: "https://developer.nvidia.com/dcgm"
+short_name: NVIDIA DCGM
+long_name: NVIDIA Data Center GPU Manager (DCGM)
+logo_path: /stackdriver/images/integrations/nvidia.png # supplied by google technical writer
+description: |-
+  The NVIDIA Data Center GPU Manager (DCGM) integration collects advanced GPU metrics,
+  including SM block utilization, Pipe utilization, PCIe and NVLink traffic.
+configure_integration: |-
+  You must install DCGM and run the DCGM daemon service.
+supported_operating_systems: linux
+supported_app_version: ["3.1"]
+gpu_platforms: # p4, p100 don't emit DCGM profiling metrics
+  - model: a100
+    platforms:
+      - ubuntu-os-cloud:ubuntu-2004-lts
+  - model: v100
+    platforms:
+      - ubuntu-os-cloud:ubuntu-2004-lts
+  - model: t4
+    platforms:
+      - ubuntu-os-cloud:ubuntu-2004-lts
+  - model: l4
+    platforms:
+      - debian-cloud:debian-11
+      - ml-images:common-gpu-debian-11-py310
+      - rocky-linux-cloud:rocky-linux-8
+      - rocky-linux-cloud:rocky-linux-9
+      - suse-cloud:sles-15
+      - ubuntu-os-cloud:ubuntu-2004-lts
+      - ubuntu-os-cloud:ubuntu-2204-lts
```

@LujieDuan Replicating [your comment](https://github.com/GoogleCloudPlatform/ops-agent/pull/1758/files/98ac11fd7f47de4ed6073d64f90c105a75f1c9e7#r1763717461) here.
```suggestion
      - ubuntu-os-cloud:ubuntu-2204-lts
  - model: p100
    platforms:
      - ubuntu-os-cloud:ubuntu-2004-lts
  - model: p4
    platforms:
      - ubuntu-os-cloud:ubuntu-2004-lts
```
With the new receiver, P4 and P100 now can send DCGM metrics, so we should enable the DCGM test on those two GPUs.

---

Unfortunately, most profiling metrics are actually not available on P4 and P100, so I've had to revert this change.